src/forms/Slushbucket.tsx

```diff
@@ -0,0 +1,237 @@
+import React, { Component } from "react";
+
+import MDSpinner from "react-md-spinner";
+import Scrollbars from "react-custom-scrollbars";
+import ClearButton from "../components/ClearButton.js";
+
+import styles from "./SlushBucket.css";
+import formStyles from "../styles/Forms.css";
+import inputStyles from "../styles/Input.css";
+import displayStyles from "../styles/Display.css";
+
+type choiceT = {value: string | number, display: string};
+export type choicesT = [choiceT];
+type valueItemT = string | number
+type valueT = [valueItemT];
+
+interface Props {
+  title: string,
+  name: string,
+  placeholder: string,
+  readonly: boolean | undefined,
+  isFetching: boolean | undefined,
```

Comment:
(switching to English, forgot earlier)

Optional properties can be defined as isFetching?: boolean.

Reply:
But this one isn't needed.